easybuild/tools/filetools.py

```diff
@@ -792,6 +792,57 @@ def find_easyconfigs(path, ignore_dirs=None):
     return files
 
 
+def locate_files(files, paths, ignore_subdirs=None):
+    """
+    Determine full path for list of files, in given list of paths (directories).
+    """
+    # determine which easyconfigs files need to be found, if any
+    files_to_find = []
+    for idx, ec_file in enumerate(files):
+        if ec_file == os.path.basename(ec_file) and not os.path.exists(ec_file):
+            files_to_find.append((idx, ec_file))
+    _log.debug("List of files to find: %s" % files_to_find)
+
+    # find missing easyconfigs by walking paths in robot search path
+    for path in paths:
+        _log.debug("Looking for missing files (%d left) in %s..." % (len(files_to_find), path))
+
+        # try to load index for current path, or create one
+        path_index = load_index(path, ignore_dirs=ignore_subdirs)
+        if path_index is None or build_option('ignore_index'):
+            if os.path.exists(path):
```

Comment:
This would probably be better to have before the log_debug, i.e. first check if the path exists at all and if not just skip it completely, i.e. "if not os.path.exists(path): next" or however that is done in python

Reply:
Very good point, fixed in bb18de4